src/handlers/MessageCommandHandler.ts

```diff
@@ -152,6 +153,19 @@ export async function MessageCommandHandler(
 	}
 
 	if (!command.type.includes(CommandType.MESSAGE)) return;
+	if (!message.guild && command.dmPermission === false) return;
+	if (
+		command.defaultMemberPermissions &&
+		!command.inhibitors?.some(
+			inhibitor => inhibitor.constructor.name !== 'MemberPermissions',
+		)
+	) {
+		command.inhibitors.push(
```

Comment:
This will add the inhibitor for slash commands as well, which is not needed. Maybe you can run this inhibitor separate? So dont add it to the commands inhibitors?

Reply:
ah yes!